scripts/ci/update-github

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -ex
+
+TAG_NAME=$(git tag -l --contains HEAD)
+
+rm -rf dcos-nightly
+
+# Setup Git
+git config --global user.email "$GIT_USER@users.noreply.github.com"
+git config --global user.name "MesosphereCI Robot"
+git config credential.helper 'cache --timeout=300'
+git clone "https://$GIT_USER:$GIT_PASSWORD@github.com/mesosphere/dcos" dcos-nightly
+cd dcos-nightly
+
+if [[ $TAG_NAME ]]
+then
+  git checkout -- .
+  git remote add upstream "https://$GIT_USER:$GIT_PASSWORD@github.com/dcos/dcos"
+  git checkout master
+  git checkout -b dcos-ui/$TAG_NAME
+
+  cp -fR ../buildinfo.json packages/dcos-ui/buildinfo.json
```

Comment:
I think we don't need the `-R` here as it is just one file to copy (and it was a bit confusing to me, I went ahead to find out why `buildinfo.json` should be a directory